interpreter/test/call_indirect.wast

```diff
@@ -360,3 +360,12 @@
   )
   "unknown type"
 )
+
+;; invalid table
+(assert_invalid 
+  (module
+    (table anyfunc (elem 0 $t1 44))
+    (func $t1 (result i32) (i32.const 1))
```

Comment:
Ah, I actually meant the entire function, not just the name. AFAICS, the minimal test would be just
```
(module (table anyfunc (elem 0 0)))
```